app/build.gradle

```diff
@@ -34,6 +34,7 @@ android {
 
     release {
       debuggable false
+      useProguard false
```

Comment:
We don't need this here -- It's already present in gradle.properties: https://github.com/resolvetosavelives/redapp/blob/master/gradle.properties